inference/conflict.go

```diff
@@ -149,7 +152,8 @@ func (l *conflictList) addSingleAssertionConflict(pass *analysis.Pass, trigger a
 	l.conflicts = append(l.conflicts, c)
 }
 
-func (l *conflictList) addOverconstraintConflict(nilExplanation ExplainedBool, nonnilExplanation ExplainedBool, pass *analysis.Pass) {
+// AddOverconstraintConflict adds a new overconstraint conflict to the list of conflicts.
+func (l *ConflictList) AddOverconstraintConflict(nilExplanation ExplainedBool, nonnilExplanation ExplainedBool, pass *analysis.Pass) {
```

Comment:
Nothing currently calls this from outside this package. This is just done for consistency with `AddOverconstraintConflict` being visible. Correct? (Good by me, just want to make sure)

Reply:
That's right. :)

Reply:
Plus once we move the diagnostic processing logic to a separate package we'll need this exported

Reply:
Good point :)